src/plugins/intel_npu/src/plugin/src/backends.cpp

```diff
@@ -109,9 +110,10 @@ NPUBackends::NPUBackends(const std::vector<AvailableBackends>& backendRegistry,
             }
 #endif
         } catch (const std::exception& ex) {
-            _logger.error("Got an error during backend '%s' loading : %s", backendName.c_str(), ex.what());
+            _logger.warning("Got an error during backend '%s' loading : %s", backendName.c_str(), ex.what());
+            // todo: can we get detail exception type and then throw it? need check openvin_throw
```

Comment:
todo: can we get detail exception type and then throw it? need create a exception class (where?)